test/test_que_locks.rb

```diff
@@ -283,11 +283,17 @@ def test_multiple_enqueued_untouched_active_jobs
   end
 
   def test_can_enqueue_locked_active_job
+    key = "_aj_symbol_keys"
+
+    if Gem::Version.new(Que::VERSION) >= Gem::Version.new("2.0")
```

Comment:
I'm a little confused. This keyword comes from ActiveJob, not from Que, right? Can you explain why we're deciding which one to use based on the Que version?

Reply:
Yes, I wasn't thinking straight when I did this and realised later. I will update this to be based on the active job version that changed the serialization.